### DISCUSSION TITLE
Contributor Motivations/Profiles

### DISCUSSION THREAD

**Discussion by doubledh** [Imaginary-Horizons-Productions/prophets-of-the-labyrinth-Archive#413] [Show and tell]
Normally, contributors may self-serve the issues they work on.
However, if contributors would like recommendations or prioritizations on Issues, they should be able to ask the manager/leadership for recommendations on issues within the contributors' individual interests.

Therefore, I am hosting this thread to get an idea about each of our contributors:
1) what do you look forward to _gaining_ from contributing?
  - like experience, practice, cooperation, friendship, portfolio, money, fame, etc. 
2) what _skills_ do you offer to the team?
  - specific proficiencies of creation, so I can think to call on those talents if we need them and you offer them
  - ex: able to write music, able to draw graphics, able to code in JS or use certain frameworks/tools 
3) how much _time_ do you offer to the team?
  - Maybe you're busy with a career and are trying to squeeze in this as a side project, no biggie
  - maybe you have free time AND you'd like to spend it all working with us 🤷‍♂️(serendipitous!) 
4) what _kind_ of contributions are you looking to make, or avoid?
  - maybe you like making silly characters/mobs, or maybe you like making story/lore
  - perhaps you hate SQL
  - 
5) what do you see PotL as? 
  - (super open-ended question) 
  - could come into conflict with the vision, but could also improve/guide the vision
  - Do you prefer solving high level/abstract/architecture or low-level/concrete/coding problems?
 
  As a manager, I will take the responses as feedback, to 
  - involve other contributors in ways that suit their individual interests
  - protect contributors' interests when it appears that multiple interests may inevitably conflict
  
  That said, this thread should not be used as a place to assert superiority or inferiority between contributors. Whoever's here, willing, and able to respect others' interest, will be welcome to contribute as they please and to their own ability

**Comment by doubledh** [comment]
I'll start of with my own responses:

1. I'm looking forward to 
  - learning more JS
  - getting experience from a ground-up developer point of view
  - have a side-project
  - be able to work on a game (childhood dream; never put forth a whole-hearted effort)
 
 2) The skills I offer to the team
  - Software engineer at SAP Concur; I pipe data from and to databases
  - Node JS 
  - SQL (mysql/postgres/sqlserver) and NoSQL (phoenix/dynamodb)
  - I've handled quite a few race-condition conundrums, and am otherwise quite good at bug fixing
  - worked with logging/CICD/container/cloud platforms 
  - linux/bash 
  - Novice at music software and composition
 
 3) Time available: I work in CT timezone on weekdays, have activities on weekends and weeknights, but am willing to double-book some nights. Mostly available on Wed-Sat 

 4) I'm happy to contribute anything, really. Probably would prefer to just do more important work, and I know I have a knack (or at least tolerance :P)  for bug-fixing. I think I do dislike handling network config, though.

 5) I see PotL as an accessible, portable, and novel bot that can provide me with a straightforward, dynamic experience that I can enjoy alongside the cooperation of friends

**Comment by ntseng** [comment]
### 1) what do you look forward to _gaining_ from contributing?
- Practice implementing ideas
- An end-product I can be proud of; a game that's legitimately fun and brings people together
- Some innovation: I'm looking forward to seeing how far we can take this concept of *multiplayer forecasting roguelite*
### 2) what _skills_ do you offer to the team?
- Being a generalist
- I have some professional game design experience
### 3) how much _time_ do you offer to the team?
- Developing side-projects like PotL is my hobby
### 4) what _kind_ of contributions are you looking to make, or avoid?
- Any and every thing, if you want something make sure to beat me to calling dibs (or we can collaborate)!
### 5) what do you see PotL as?
- A hobby: I'm not rushing to add the pressure of profit or fame to the project any time soon
- However, I also want PotL to be rewarding (even if not monetarily) for everyone who contributes to it

**Comment by bobaguardian** [comment]
I think it's really cool you're doing this!

1. What do I look forward to gaining from contributing
- Getting more experience as a dev in general (working with a team, reading others' code)
- Working on a side project to keep my skills fresh - I love games and I think this is a really cool project
- I use discord a lot, so I thought this would be a great way to maybe implement a bot of my own in the future
- I like to separate my work time and me time to avoid burnout and right now I'm working on PotL in a pace where it's considered me time and I'd like to keep it that way :)

2. What skills do I offer to the team?
- I'm a fresh dev out of App Academy's bootcamp (JavaScript, React/Redux, Python, PostgreSQL)
- I usually main support in games and it translates over to my work -- Hoping to just be there for dev or playtest support in general -- If you guys are the Giovanni, I'll gladly be the Team Rocket Grunts

3. How much time do you offer to the team?
- Currently in the job hunt under App Academy's timeline/structure so the amount of time really depends on how my interviews are going (i.e. lots of interviews lined up or not) - so it varies!
- I'm trying to use PotL as a buffer to my day to day responsibilities on the App Academy career quest
- I'm on the PST timezone (based in San Jose, CA)

4. What kind of contributions are you looking to make, or avoid?
- I'm happy to contribute to anything really.  I don't mind bug fixing so if we need more of that, I'm down.  
- I'm usually more inclined to pick up issues I know I can do/have enough info/knowledge about, but if there's something I'd like to work on, I won't hesitate to ask more questions/clarify :) 

5.  What do you see PotL as?
- A cool side project to work on with some friends!  
- When I was onboarded by Nathaniel, I remember being amazed with what we had so far and I'd love to eventually enjoy this game with my other friends and have it be something I'm proud to have contributed to.

**Reply by ntseng** [reply]
Yeah, I definitely had to take "game dev breaks" from job searching too. When you figure out your favorite things to work on, let us know so we can have that available!
